WorkloadManagementSystem/PilotAgent/pilotTools.py

```diff
@@ -31,7 +31,7 @@ def printVersion( log ):
 def pythonPathCheck():
 
   try:
-    os.umask( 18 ) # 022
+    os.umask( 0o22 )
```

Comment:
Even though I like this style much more, Andrei changed it in another PR because it was not understood by some old python version...